src/redux/features/tracking/trackingSlice.ts

```diff
@@ -24,9 +24,31 @@ export const trackingSlice = createSlice({
 });
 
 export const { setPlantingSitesAction } = trackingSlice.actions;
-
 export const trackingReducer = trackingSlice.reducer;
 
+// Define a type for the search slice state
+type SearchData = {
+  sites?: PlantingSiteSearchResult[];
+};
+
+// Define the initial state for search
+const initialSearchState: SearchData = {};
+
+export const plantingSitesSearchResultsSlice = createSlice({
+  name: 'plantingSitesResultsSlice',
+  initialState: initialSearchState,
+  reducers: {
+    setSearchPlantingSitesAction: (state, action: PayloadAction<SearchData>) => {
```

Comment:
one request, there's a mix of PlantingSitesSearch and SearchPlantingSites across the code. We could just make this consistent PlantingSitesSearchResults